frontend/src/app/modules/home/issues/issues.component.ts

```diff
@@ -42,14 +54,19 @@ export class IssuesComponent extends BaseComponent implements OnInit {
         event.originalEvent.stopPropagation();
     }
 
+    private showPaginator() {
+        this.paginators = this.numberOfIssues > this.itemsPerPage;
+    }
+
     private loadIssues() {
-        this.issueService.getIssuesInfo()
-            .pipe(this.untilThis)
-            .subscribe(issues => {
-                this.issues = issues;
-            }, errorResponse => {
-                this.toastNotification.error(errorResponse, 'Error', 1500);
-            });
+        return this.issueService.getIssuesInfo()
+            .pipe(
+                tap(issues => {
+                    this.issues = issues;
+                    this.numberOfIssues = issues.length;
```

Comment:
Why do you need this? You can just use issues.length all the time

Reply:
Fix